contrib/freshmaker-cli

```diff
@@ -225,6 +249,48 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, days=7):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL and the issue date is between specified
+    dates.
+    """
+    if days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, auth=krb_auth, timeout=30)
```

Comment:
This doesn't perform pagination. I thought we wanted to do that to ensure all advisories in the specified date range exists.

Reply:
Pagination can be done with simple helper function:

```python
def get_paginated_items(session, url):
    page = 0
    while True:
        page += 1
        r = session.get(url, params={page: page})
        r.raise_for_status()
        data = r.json()
        if not data:
            break
        for advisory in data:
            yield data()
```